### PR TITLE
Mon 3370 refactor existing ui elements

### PR DESCRIPTION
**Pull request by NikolaL87**

### CODE REVIEW

File: lib/Checkbox/checkbox.scss [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        top: -1px;
+        left: -1px;
+        display: block;
+        width: 15px;
+        height: 15px;
+        border-radius: 0;
+        content: "";
+      } 
+      &:before {
+        background: url("../../img/icons/checkbox-buttons-stripe.png") no-repeat;
```
Comment (EdytaSki) [inline lib/Checkbox/checkbox.scss]: Please use the svg formt of images example background-image: url("../../img/icons.checkbox-buttons-stripe.svg")
test on cross browser env

File: lib/Checkbox/checkbox.scss [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        background-position: -8px -16px;
+      }
+    }
+  }
+}
+
+input[type=checkbox]:checked~.custom-control-label::after {
+  background: url("../../img/icons/checkbox-buttons-stripe.png") no-repeat;
```
Comment (EdytaSki) [inline lib/Checkbox/checkbox.scss]: svg images please

File: src/Checkbox/checkbox.scss [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        top: -1px;
+        left: -1px;
+        display: block;
+        width: 15px;
+        height: 15px;
+        border-radius: 0;
+        content: "";
+      } 
+      &:before {
+        background: url("../../img/icons/checkbox-buttons-stripe.png") no-repeat;
```
Comment (EdytaSki) [inline src/Checkbox/checkbox.scss]: Images in svg please - we should be able to change colors

File: src/Checkbox/checkbox.scss [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+        background-position: -8px -16px;
+      }
+    }
+  }
+}
+
+input[type=checkbox]:checked~.custom-control-label::after {
+  background: url("../../img/icons/checkbox-buttons-stripe.png") no-repeat;
```
Comment (EdytaSki) [inline src/Checkbox/checkbox.scss]: svg image please

File: src/RadioButton/radio-button.scss [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+      left: 0;
+      display: block;
+      width: 15px;
+      height: 15px;
+      content: "";
+      top: -1px;
+      border-radius: 50%;
+    }
+    &:before {
+      background: url("../../img/icons/radio-buttons-stripe.png") no-repeat;
```
Comment (EdytaSki) [inline src/RadioButton/radio-button.scss]: background svg please

File: src/RadioButton/radio-button.scss [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+    }
+    &:after {
+    background-repeat: no-repeat;
+    background-position: 50%;
+    background-size: 50% 50%;
+    }
+  }
+}
+.custom-radio .form-check-input:checked~.custom-control-label:after {
+  background: url("../../img/icons/radio-buttons-stripe.png") no-repeat;
```
Comment (EdytaSki) [inline src/RadioButton/radio-button.scss]: BG - SVG

File: src/Title/custom-title.scss [hunk 7]
```diff
@@ -21,15 +21,20 @@
...
     width: 51px;
     height: 51px;
-    display: inline-block;
-    vertical-align: top;
-    margin-right: 7px;
+    position: absolute;
+    left: 0;
+    top: 0;
     &-puzzle {
       background: url('../../img/icons/icon-title-puzzle.png') no-repeat 50%;
```
Comment (EdytaSki) [inline src/Title/custom-title.scss]: images in svg please

**Comment by NikolaL87** [comment]
Changes are related to the task: 
MON-3386 - Transform all icons and symbols from .png/.jpg format to SVG

**Comment by EdytaSki** [comment]
We close this PR because we have a new JIRA task to continue on this refacto.

**Pull request merged**